src/main/java/org/apache/cassandra/sidecar/utils/SSTableImporter.java

```diff
@@ -623,4 +645,38 @@ public ImportOptions build()
             }
         }
     }
+
+    /**
+     * Key used for the map of queues
+     */
+    protected static class ImportKey
+    {
+        private final String host;
+        private final String keyspace;
+        private final String table;
+
+        public ImportKey(String host, String keyspace, String table)
+        {
+            this.host = host;
+            this.keyspace = keyspace;
+            this.table = table;
+        }
+
+        @Override
+        public boolean equals(Object o)
+        {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            ImportKey importKey = (ImportKey) o;
+            return Objects.equals(host, importKey.host)
+                   && Objects.equals(keyspace, importKey.keyspace)
+                   && Objects.equals(table, importKey.table);
+        }
+
+        @Override
+        public int hashCode()
+        {
+            return Objects.hash(host, keyspace, table);
```

Comment:
NIT: The object is only going to be used as map keys. We can store the hashcode as an instance variable. (so compute only once)